### PR TITLE
- mount special filesystems in target during installation

### PR DESCRIPTION
**Pull request by aschnell**
For https://trello.com/c/NQ524dss/459-8-storageng-bootloader-proposal-efi.

### CODE REVIEW

**Comment by shundhammer** [comment]
LGTM

File: src/lib/y2storage/clients/inst_prepdisk.rb [hunk 1]
```diff
@@ -51,6 +54,24 @@ def commit
...
+        mount_in_target("/dev", "devtmpfs", "-t devtmpfs")
+        mount_in_target("/proc", "proc", "-t proc")
+        mount_in_target("/sys", "sysfs", "-t sysfs")
+      end
+
+      def mount_in_target(path, device, options)
+        storage = Y2Storage::StorageManager.instance
+        target_path = storage.prepend_rootprefix(path)
+
+        if !Yast::FileUtils.Exists(target_path)
```
Comment (ancorgs) [inline src/lib/y2storage/clients/inst_prepdisk.rb]: Is not enough with Ruby's `FileUtils`? At some point we would like to see `Yast::Fileutils` die.

File: src/lib/y2storage/clients/inst_prepdisk.rb [hunk 2]
```diff
@@ -26,13 +26,16 @@
 
 Yast.import "SlideShow"
 Yast.import "Installation"
+Yast.import "FileUtils"
+Yast.import "String"
```
Comment (ancorgs) [inline src/lib/y2storage/clients/inst_prepdisk.rb]: Why is `Yast::String` needed?

File: src/lib/y2storage/clients/inst_prepdisk.rb (hunk 1 above)
Comment (aschnell) [inline src/lib/y2storage/clients/inst_prepdisk.rb]: No idea, I didn't try it. I asked on IRC what is the best way now but didn't get any reply.

File: src/lib/y2storage/clients/inst_prepdisk.rb (hunk 2 above)
Comment (aschnell) [inline src/lib/y2storage/clients/inst_prepdisk.rb]: Indeed, it's not needed.

**Pull request merged**